courses/machine_learning/deepdive2/text_classification/labs/reusable_embeddings.ipynb

```diff
@@ -9,6 +9,8 @@
     "**Learning Objectives**\n",
```

Comment:
<p>Maybe we could make a TODO here. For example, have them use the result of the signature printed above to fill in the "keras_layer_1_input" part? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>